### PR TITLE
Update running k6

### PR DESCRIPTION
**Pull request by ppcano**
Running cloud tests has been integrated into `Running k6`.

Screen Shot 2020-05-25 at 2 25 34 AM

Screen Shot 2020-05-25 at 2 27 56 AM

Full instructions for Running cloud tests has been moved to the k6 Cloud docs.  

Screen Shot 2020-05-25 at 2 25 52 AM

Available in `staging`.

### CODE REVIEW

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md [hunk 1]
```diff
@@ -221,63 +181,26 @@ export default function() {
...
-  });
-  sleep(1);
-}
-```
+One of the goals with k6 is to support three execution modes to run your k6 tests:
 
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
+- Clustered: on more than one machine managed by you. Not supported yet.
```
Comment (robingustafsson) [inline src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md:190]: I'd link the "Not suppported yet" sentence to the issue in question: https://github.com/loadimpact/k6/issues/140

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md [hunk 2]
```diff
@@ -221,63 +181,26 @@ export default function() {
...
-    'transaction time OK': r => r.timings.duration < 200,
-  });
-  sleep(1);
-}
-```
+One of the goals with k6 is to support three execution modes to run your k6 tests:
 
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
```
Comment (robingustafsson) [inline src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md:189]: Why is k6 Cloud in a back-tick "code" styling?

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md [hunk 3]
```diff
@@ -221,63 +181,26 @@ export default function() {
...
+- Clustered: on more than one machine managed by you. Not supported yet.
 
-_Using check() to verify that transactions work and are fast enough_
+The goal of k6 is to support running a test on the three execution modes without making modifications to the script.
```
Comment (robingustafsson) [inline src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md:192]: This sentence just feels like a repeat of the sentence from above:
> One of the goals with k6 is to support three execution modes to run your k6 tests:

Maybe remove this sentence and add ", without making modifications to the script" to the sentence above the three bullet points?

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md [hunk 4]
```diff
@@ -221,63 +181,26 @@ export default function() {
...
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
+- Clustered: on more than one machine managed by you. Not supported yet.
 
-_Using check() to verify that transactions work and are fast enough_
+The goal of k6 is to support running a test on the three execution modes without making modifications to the script. 
 
-The above will generate a couple of extra output lines after the test, telling you if your check conditions succeeded or failed during the test. If the check conditions never failed, you will see this:
+For running cloud tests, you must sign up with the k6 Cloud and log in with the CLI. Then, you'd only have to pass your existing script to the `k6 cloud` command.
```
Comment (robingustafsson) [inline src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md:194]: I'd rephrase this sentence a bit to clarify: "For running cloud tests from the CLI, you must first register a k6 Cloud account and then log into your account via the CLI. Then, you only have to pass your existing script to the `k6 cloud` command."

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+      execution: cloud
+      script: test.js
+      output: https://app.k6.io/runs/TEST_RUN_ID
+    ```
+
+    </div>
+
+7. Navigate to the URL to check your test results.  When the test is running, the test result page is shown. 
+
+   ![Insights results](/images/cloud-insights-results.png 'k6 Cloud Test Results')
```
Comment (robingustafsson) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:96]: It's just an alt label but "Insights results" should be replaced with "k6 Cloud Test Results".

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+
+   Learn more about the different test result sections on the [k6 Cloud Results docs](/cloud/analyzing-results/overview).
+
+## Cloud execution options
+
+All the [k6 Options](/using-k6/options), like `--vus` and `--duration` are the same between the `k6 run` and `k6 cloud` commands. k6 aims to run the same script on different execution modes without making any script modifications.
```
Comment (robingustafsson) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:102]: "**on** different execution modes" => "**in** different execution modes"

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+| name (string)         | Optional. The name of the main script file, so something like "script.js".                       | The name of the test in the k6 Cloud UI. Test runs with the same name will be grouped together.                                                                                                                                     |
+| projectID (number)         | Optional. It is empty by default.                       | The ID of the project in which the test is assigned in the k6 Cloud UI. By default, the default project of the user default organization.                                                                                                                                       |
+| distribution (object) | Optional. The equivalent of `someDefaultLabel: { loadZone: "amazon:us:ashburn", percent: 100 }`. | How the traffic should be distributed. The keys are string labels that will be injected as [environment variables](/using-k6/environment-variables) on the appropriate nodes (matching the `loadZone`): `__ENV["someDefaultLabel"]` |
+
+### List of supported load zones
```
Comment (robingustafsson) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:131]: We're missing "São Paulo" (`amazon:br:sao paulo`) from this list.

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+| Hong Kong             | `amazon:cn:hong kong` |
+
+
+### Running tests under a different project than your default one
+
+By default tests and test runs will be created and run under your default project, in your default organization.
+
+To create and run tests under a different project, whether under your default organization or one you've been invited to, you have to pass the `Project ID` to k6.
+
+Select the project on the sidebar menu and you will find the `Project ID` on the header of the Project Dashboard page.
```
Comment (robingustafsson) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:159]: "**on** the header" => "**in** the header"

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+not yet supported by the cloud execution mode.
+
+
+### Using setup/teardown life-cycle functions
+
+Your [setup and teardown life cycle functions](/using-k6/test-life-cycle)
+are executed as normal when running cloud tests. Depending on the size
+of your test, it will execute from one or more cloud servers, but the
+setup and teardown will only execute from one server, so execute once
+each test-wide. There's no guarantee though that the same cloud server
```
Comment (robingustafsson) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:264]: "so execute once each test-wide" => "so execute once for each test run"

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md (hunk 2 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md:189]: Initially, it was a link. Then, I created the link to the beginning of the phrase. 

I didn't want to have two same links on the same sentence and removed that one and used `backtick`.

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md (hunk 9 above)
Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md (hunk 8 above)
Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md (hunk 7 above)
Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md:131]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md (hunk 6 above)
Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md (hunk 5 above)
Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md (hunk 4 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md (hunk 3 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md (hunk 2 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720

File: src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md (hunk 1 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md]: https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720
